CMake/SlicerGenerateSlicerConfig.cmake

```diff
@@ -21,6 +21,8 @@
 set(Slicer_USE_FILE_CONFIG ${Slicer_USE_FILE})
 
 # Compilation settings
+set(ep_common_c_flags "${CMAKE_C_FLAGS_INIT} ${ADDITIONAL_C_FLAGS}")
+set(ep_common_cxx_flags "${CMAKE_CXX_FLAGS_INIT} ${ADDITIONAL_CXX_FLAGS}")
```

Comment:
Instead let's do this:

```cmake
set(Slicer_CMAKE_CXX_FLAGS_CONFIG    ${CMAKE_C_FLAGS_INIT} ${ADDITIONAL_C_FLAGS})
set(Slicer_CMAKE_CXX_FLAGS_CONFIG    ${CMAKE_CXX_FLAGS_INIT} ${ADDITIONAL_CXX_FLAGS})
```

The variables `ep_common_c/cxx_flags` are specific to superbuild extensions  and shouldn't be part of the SlicerConfig interface

Reply:
Sounds good, done!